src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx

```diff
@@ -715,7 +715,7 @@ function getInitialPodContainerSelection(
             };
         }
 
-        let containers = new Set(_allSelectedPods.flatMap((_po) => _po.containers ?? []));
+        let containers = new Set(_allSelectedPods.flatMap((_po) => flatContainers(_po) ?? []));
```

Comment:
rename _po to something relevant like `_pod`

Reply:
Done